contrib/IECoreArnold/include/IECoreArnold/UniverseBlock.h

```diff
@@ -50,13 +50,11 @@ class IECOREARNOLD_API UniverseBlock : public boost::noncopyable
 
 	public :
 
-		/// \deprecated
-		UniverseBlock();
 		/// Ensures that the Arnold universe has been created and
 		/// that all plugins and metadata files on the ARNOLD_PLUGIN_PATH
 		/// have been loaded. If writable is true, then throws if
 		/// there is already a writer.
-		UniverseBlock( bool writable );
+		UniverseBlock( bool writable = false );
```

Comment:
I think I'd prefer not having a default argument here, and forcing the caller to think about what they need.